messaging/Controllers/SteveController.cs

```diff
@@ -12,6 +12,7 @@
 namespace messaging.Controllers
 {
     [Route("STEVE/{jurisdictionId:length(2)}/Bundle")]
+    [Route("STEVE/{jurisdictionId:length(2)}/Bundle/{vitalType:length(4)}/{igVersion}")]
```

Comment:
The {vitalType} field doesn't get used, is that intentional? Or does it have to have the same endpoint here because that endpoint exists in the BundlesController?